js/app.js

```diff
@@ -20,36 +21,44 @@ var imgElementTwo = document.getElementById('pic-two');
 var imgElementThree = document.getElementById('pic-three');
 
 function randomPicture() {
-  //we want to write an array that iterates three times to generate three random numbers that each CANNOT be the same
-
-  //we want the numbers to be drawn from the indexes of our Picture.allPictures array 
+  //we want three numbers to be drawn from the indexes of our Picture.allPictures array 
+  var indexArray = [];
+  for (var count = 0; count < 3; count++) {
+    var randomIndex = Math.floor(Math.random() * Picture.allPictures.length);
+  
+    indexArray.push(randomIndex);
+    console.log(indexArray);
+
+    while (count < 3) {
+      imgElementOne.src = Picture.allPictures[indexArray[count]].filepath;
```

Comment:
Because you're never updating the count, this loop is infinite, so the browser cannot load it.